### DISCUSSION TITLE
double payment

### DISCUSSION THREAD

**Discussion by kite-uhn** [community/community#47504] [New to GitHub] [answered]
I created this account and paid for it. First I paid for an organization account for $44. Then I realized that i have to upgrade my seat's privilege to PRO to be able to accomplish my requirements. That was $48.
So now I'm not sure if my company credit card is charged for $(48+44) or only $48?
Please note that There would be just this one user on this organization and it doesn't seem fair to pay twice for user in one organization. I prefer to downgrade this account to one single PRO user in that case.
Can you make this clear for me? I have to explain this matter to finance department and I'm completely confused now.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @kite-uhn ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by kite-uhn** [comment]
Actually I did, but after 5 days nobody care to reply my ticket from support team.

**Reply by queenofcorgis** [reply]
Hi there - tickets are answered in the order they were received based on our team's resources. We appreciate your patience - someone should be in touch soon. Thanks!

**Comment by Rehab-Osman** [comment]
I facing the same issue too :(
did they answer you?

**Comment by billygk** [comment]
Same issue with double payment, left ticket ... but no response so far. :(
Did any get an answer to this?
